pypeit/coadd2d.py

```diff
@@ -943,17 +946,32 @@ def load_coadd2d_stacks(self, spec2d, chk_version=False):
             spat_flexure_list.append(s2dobj.sci_spat_flexure)
 
             sciimg_stack.append(s2dobj.sciimg)
+            exptime_stack.append(s2dobj.head0['EXPTIME'])
             waveimg_stack.append(s2dobj.waveimg)
             skymodel_stack.append(s2dobj.skymodel)
             sciivar_stack.append(s2dobj.ivarmodel)
             mask_stack.append(s2dobj.bpmmask.mask)
             slitmask_stack.append(s2dobj.slits.slit_img(flexure=s2dobj.sci_spat_flexure))
 
+        # check if exptime is consistent for all images
+        exptime_coadd = np.percentile(exptime_stack, 50., interpolation='higher')
```

Comment:
np.percentile(interpolation='higher') is deprecated in numpy. Can you replace with the method keyword?

Reply:
And/or use the code I gave above.

Reply:
Changed to use `method`